app/components/settings/project_custom_fields/header_component.html.erb

```diff
@@ -4,30 +4,29 @@
     render Primer::OpenProject::PageHeader.new do |header|
       header.with_title(variant: :default) { t('settings.project_attributes.heading') }
       header.with_description { t('settings.project_attributes.heading_description') }
-      header.with_actions do
-        flex_layout(justify_content: :space_between, align_items: :center) do |action_buttons_container|
-          action_buttons_container.with_column(mr: 2) do
-            render(Primer::Beta::Button.new(
-              tag: :a,
-              href: new_admin_settings_project_custom_field_path(type: "ProjectCustomField"),
-              scheme: :primary,
-              data: { turbo: "false", qa_selector: "new-project-custom-field-button" }
-            )) do |button|
-              button.with_leading_visual_icon(icon: :plus)
-              t('settings.project_attributes.label_new_attribute')
-            end
+      header.with_breadcrumbs(breadcrumbs_items)
+      flex_layout(justify_content: :space_between, align_items: :center) do |action_buttons_container|
```

Comment:
Remove the flex_layout. It is not needed any more. Further, the second button is now shown at all